src/stactools/cli/commands/addasset.py

```diff
@@ -0,0 +1,92 @@
+from typing import List, Optional
+
+import click
+import pystac
+
+from stactools.core import add_asset_to_item
+
+
+def _add_asset(
+    item_path: str,
+    asset_key: str,
+    asset_path: str,
+    title: Optional[str] = None,
+    description: Optional[str] = None,
+    media_type: Optional[str] = None,
+    roles: Optional[List[str]] = None,
+    move_assets: bool = False,
+    ignore_conflicts: bool = False,
+) -> None:
+    item = pystac.read_file(item_path)
+    if not isinstance(item, pystac.Item):
+        raise click.BadArgumentUsage(f"{item_path} is not a STAC Item")
+    asset = pystac.Asset(asset_path, title, description, media_type, roles)
+    item = add_asset_to_item(
+        item,
+        asset_key,
+        asset,
+        move_assets=move_assets,
+        ignore_conflicts=ignore_conflicts,
+    )
+    item.save_object()
+
+
+def create_addasset_command(cli: click.Group) -> click.Command:
+    @cli.command("addasset", short_help="Add an asset to an item.")
```

Comment:
We should establish a convention for multi-word commands. We currently only have two cases: `create-item` and `addraster`. I think we should standardize on kebab-case, and change addraster. @gadomski

Reply:
👍🏽 to kebab case, ticket created: https://github.com/stac-utils/stactools/issues/322.